__tests__/app.test.js

```diff
@@ -353,4 +431,43 @@ describe("my express project", () => {
         });
     });
   });
+
+  describe("DELETE: /api/comments/:comment_id", () => {
+    it("204 deletes given comment by comment_id", () => {
+      const id = 4;
+      return request(app)
+        .delete(`/api/comments/${id}`)
+        .expect(204)
+        .then(() => {
+          return db
+            .query(`SELECT COUNT(comment_id)::INT FROM comments`)
+            .then(({ rows }) => {
+              const comment_count = rows[0].count;
+              expect(comment_count).toBe(17);
+            });
```

Comment:
this query is correct but it's not clear from reading it that it's correct (from only reading the test i don't know if `17` is the expected number)

If i was on your team i'd have go to the testdata and count the files then subtract 1 (which is a lot of work 😅)

It would be clearer if you were making a request for that particular comment and asserted that it wasn't found